src/kernel/chainstatemanager_opts.h

```diff
@@ -42,7 +42,6 @@ struct ChainstateManagerOpts {
     std::optional<uint256> assumed_valid_block{};
     //! If the tip is older than this, the node is considered to be in initial block download.
     std::chrono::seconds max_tip_age{DEFAULT_MAX_TIP_AGE};
-    DBOptions block_tree_db{};
```

Comment:
Nice! Makes a lot more sense to me to have block tree database options in the `BlockManagerOptions`